mountinfo/mountinfo.go

```diff
@@ -29,35 +29,38 @@ type Info struct {
 	// ID is a unique identifier of the mount (may be reused after umount).
 	ID int
 
-	// Parent indicates the ID of the mount parent (or of self for the top of the
-	// mount tree).
+	// Parent is the ID of the parent mount (or of self for the root
+	// of this mount namespace's mount tree).
 	Parent int
 
-	// Major indicates one half of the device ID which identifies the device class.
-	Major int
+	// Major and Minor are the major and the minor components of the Dev
+	// field of unix.Stat_t structure returned by unix.*Stat calls for
+	// files on this filesystem.
+	Major, Minor int
 
-	// Minor indicates one half of the device ID which identifies a specific
-	// instance of device.
-	Minor int
-
-	// Root of the mount within the filesystem.
+	// Root is the pathname of the directory in the filesystem which forms
+	// the root of this mount.
 	Root string
 
-	// Mountpoint indicates the mount point relative to the process's root.
+	// Mountpoint is the pathname of the mount point relative to the
+	// process's root directory.
 	Mountpoint string
 
-	// Options represents mount-specific options.
+	// Options is per mount options, comma-separated.
```

Comment:
```suggestion
	// Options are per mount options, comma-separated.
```

or perhaps

```suggestion
	// Options represents per mount options, comma-separated.
```

or `s/represents/stores/` or `s/represents/holds/` ?

Reply:
I don't like the term "represents" here (as it implies some kind of conversion).

I don't like "stores" or "holds", as this is a string, and yes, it stores some value but it's kinda obvious.

Brevity is good here.

Reply:
see my other comment, I think we can use the same approach here;

```suggestion
	// Options is a comma-separated list of per mount options.
```

Could perhaps omit the `per` here (?);

```suggestion
	// Options is a comma-separated list of mount options.
```